src/background/environment.js

```diff
@@ -11,4 +11,7 @@ export async function initializeEnvironment() {
     await browser.experiments.logins
       .setLoginSavingEnabled(origin, false);
   }
+
+  const mgmtURI = browser.extension.getURL("/list/manage.html?filter=%DOMAIN%");
```

Comment:
Might be good to document the fact that the internal desktop code replaces `%DOMAIN%` when `LoginHelper.openPasswordManager()` is called

Reply:
very good call; especially since that's not hooked up yet (follow on issue, or attempt to apply here?)

Reply:
not sure, looks like it's hooked up as of 67, at least https://bugzilla.mozilla.org/show_bug.cgi?id=1529729

Reply:
by "not hooked up yet", I meant here in the extension ... at the time I wrote that comment (-: